pkg/appconsts/chain_ids.go

```diff
@@ -6,6 +6,8 @@ const (
 	MainnetChainID = "celestia"
 	// TestChainID is the chain ID used for testing.
 	TestChainID = "test"
+	// LocalDevnetChainID is another chain ID used for testing.
+	LocalDevnetChainID = "local_devnet"
```

Comment:
Why do we need to add support for another testnet chain?

Reply:
This chain id is used by the `local_devnet` directory. The `local_devnet` directory could be refactored to use `test` so IMO we can do that in a follow up and then remove it from here.

Ref: https://github.com/celestiaorg/celestia-app/blob/8f9b3dcab883046eaca060d6aebdee5337bc4ebd/local_devnet/scripts/start_node_and_create_validator.sh#L47